test/database.yml.example

```
@@ -5,8 +5,8 @@ mysql:
   database: test
   encoding: utf8
 
-redshift:
-  adapter: redshift
+postgresql:
```

nit: Redshift に postgresql adapter で接続しているという設定なので、こっちまで変えるのはやりすぎなような。

```suggestion
redshift:
```

この変更を採用するのであれば追従して他ファイルも変更が必要です。

---

たしかにそうですね、https://github.com/bricolages/redshift_connector/pull/27/commits/9cc8f6cbae62b02b96dfa1f7ef8271918f83b6a7 で元に戻しました。